GitCommands/RevisionReader.cs

```diff
@@ -459,9 +433,14 @@ public StringLineReader(in ReadOnlySpan<char> s)
                     ? bodySlice.ToString().Replace('\v', '\n')
                     : null;
             }
-        }
 
-        #endregion
+            void ParseAssert(string message)
+            {
+                _noOfParseError++;
+                Debug.Assert(_noOfParseError > 1, message);
```

Comment:
Please use this only if a debugger is attached.

Reply:
also changing the other occurrence - affects other files too